src/hotspot/cpu/aarch64/aarch64_sve.ad

```diff
@@ -229,7 +229,6 @@ source %{
       case Op_LoadVectorGather:
       case Op_StoreVectorScatter:
       case Op_VectorLoadConst:
-      case Op_VectorRearrange:
```

Comment:
The min vector length for `VectorRearrange/VecorLoadShuffle` is 4, which decides the min vector length for byte is 4. However, currently it doesn't have such size limitation for SVE. This makes the` rearrange` tests of `Double128VectorTests/Long128VectorTests ` crash when the compiler tries to make the byte vector type with length 2. Could you please fix this issue together with this patch? Thanks so much!

Reply:
I think you can simply fix as what NEON does, like:

+      case Op_VectorLoadShuffle:
+      case Op_VectorRearrange:
+        if (vlen < 4) {
+          return false;
+        }

When we have all codegen ready, I will create another patch to fine-tune the supported vector size for each Op to get best performance.

Reply:
Yes, I have fixed in this form and tested. I think that full-test will be completed soon.